pipeline-adapters/mlrun-pipelines-kfp-v1-8/src/mlrun_pipelines/ops.py

```diff
@@ -257,6 +259,22 @@ def add_function_node_selection_attributes(
     return container_op
 
 
+def _enrich_node_selector_from_project(function):
+    function_node_selector = getattr(function.spec, "node_selector") or {}
+    project = function._get_db().get_project(function.metadata.project)
```

Comment:
I'm contemplating if we should use `get_current_project` instead because I prefer not to bloat  the SDK code with get project requests. On the other hand, we want to enrich with the values from the DB and not with cache or do we 🤔